libsolidity/formal/CHC.cpp

```diff
@@ -1011,27 +1047,30 @@ smtutil::Expression CHC::predicate(Predicate const& _block)
 
 smtutil::Expression CHC::predicate(FunctionCall const& _funCall)
 {
-	/// Used only for internal calls.
-
 	auto const* function = functionCallToDefinition(_funCall);
 	if (!function)
 		return smtutil::Expression(true);
 
+	auto getContractAddressValue = [this](FunctionCall const& _f)
```

Comment:
```suggestion
	auto contractAddressValue = [this](FunctionCall const& _f) {
```
As part of the style we don't use `get` and `set`

Reply:
What about the opening brace? Should it be on this line as in your suggestion? In other cases the opening brace is on the new line, no?

Reply:
In classes and functions yes, but in this case I think on the same line, like with operators

Reply:
OK, updated.